src/app/public/modules/autocomplete/types/autocomplete-default-search-function-options.ts

```diff
@@ -8,5 +8,4 @@ import {
 export interface SkyAutocompleteDefaultSearchFunctionOptions {
   propertiesToSearch?: string[];
   searchFilters?: SkyAutocompleteSearchFunctionFilter[];
-  searchResultsLimit?: number;
```

Comment:
This type is exported via the public API. Possible to avoid removing this value?

Reply:
Ya. See my comment on the default function.

Reply:
Added back